.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+defaults:
+  run:
+    shell: bash
```

Comment:
Is this needed?

Reply:
I've found so. otherwise the cache step runs in powershell and seems to break.